### ISSUE TITLE
Implement CEUS (NGA-East) site amplification model

### ISSUE THREAD

**Issue by pmpowers-usgs** [usgs/nshmp-haz#257]
Reference implementations supplied by Jon Stewart on 8/7/17.

**Comment by pmpowers-usgs** [comment]
Preliminary implementation complete. Waiting on guidance from Stewart et al. for very soft soils (Vs30<200 m/s) and very short periods (T<0.75s). Currently if Vs30<200, Vs30 is set to 200 m/s. For Vs30>2000, Vs30=3000 m/s. For short periods, no site (de)amplification is applied.

In going from vs30 = 3000 to 2000 m/s, short period amplification of seed models is strong.

**Comment by pmpowers-usgs** [comment]
Closed with PR #262

**Comment by pmpowers-usgs** [comment]
SP16 needs siteamp support.

**Comment by pmpowers-usgs** [comment]
Analysis suggests something is wrong with implementation. Turns out nonlinear coefficient `c` should be negative: [coeffs](/usgs/nshmp-haz/blob/master/src/gov/usgs/earthquake/nshmp/gmm/coeffs/nga-east-usgs-siteamp.csv)

**Comment by pmpowers-usgs** [comment]
Have received updated/expanded spectral period and site guidance from J. Stewart et al.

**Comment by pmpowers-usgs** [comment]
Coefficient precision is slightly different across the original and updated models and associated files and scripts. Generally rounding vs30 values to whole integers; others are largely consistent with the precision indicated in PEER report 2017/05.

**Comment by pmpowers-usgs** [comment]
Received guidance from J. Stewart regarding applicability of model down to Vs30=150m/s:

> The model is basically flat at low Vs30, so it won't do anything strange as you go below 200 m/s. We just don't have any information there, so we don't know if it's correct. As shown in Fig 2.1 of the original report, the model already captures increasing uncertainty as Vs30 decreases, so that part is taken care of. As a result, if you need to extrapolate the model for soft sites, it's probably fine to do so. 

> Some members of the panel were insistent on the 200 m/s bound, mostly because they want to require site-specific analysis for soft sites. For this reason, I may not be able to get a statement from the panel blessing the model for lower VS30.

The model should be implemented down to 150m/s and if the case is made that we should set a floor of 200m/s, then we can re-enable this behavior.

**Comment by pmpowers-usgs** [comment]
Currently implemented as scaling of median ground motion, not scaled and summed PoE curves.

**Comment by pmpowers-usgs** [comment]
An update to the NGA-East site amp model was supplied by the Stewart group on 2-23-2018 that needs to be implemented.
